temporal-sdk/src/main/java/io/temporal/common/converter/EncodingKeys.java

```diff
@@ -22,8 +22,9 @@
 import com.google.protobuf.ByteString;
 import java.nio.charset.StandardCharsets;
 
-class EncodingKeys {
-  static final String METADATA_ENCODING_KEY = "encoding";
+public class EncodingKeys {
+  public static final String METADATA_ENCODING_KEY = "encoding";
```

Comment:
This is a mess now. Why `METADATA_ENCODING_RAW` is package-private, while `METADATA_ENCODING_JSON` is made public?
Also, this class is not intended for our users, at least now. So, if it's public - it should be in *.internal.* package.

There are three ways:
1. Don't mess with it. Keep this class package-private and use the text constants directly in tests.
2. Make everything public in this class and move it to *.internal.* package
3. Don't mess with it. Keep this class package-private. In unit tests add a class in io.tempotal.common.converter the only purpose of which is to expose these constants with public modifier.

I would just go with 1. Not a big sin taking into account very small amount of tests where we need to use this constants in.